docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,292 @@
+.. _drift_user_guide:
+
+====================
+Drift User Guide
+====================
+
+**In This Guide, You Can Learn:**
+
+* `What Is Distribution Drift and How Can You Detect It? <#what-is-distribution-drift>`__
+* `How Can You Use Deepchecks to Detect Drift? <#how-can-i-use-deepchecks-to-detect-drift>`__
+
+
+What Is Distribution Drift?
+==========================
+
+Distribution drift is a change in the underlying distribution of the data, the label or of the relationship between them. Drift happens over time,
+and can occur either gradually or suddenly.
+
+Common reasons for drift can be natural changes in the data, such as shopping habits changing for an online retailer
+(for example, during the covid pandemic, virtual shopping has risen significantly and buyer patterns have changed), or
+data acquisition and processing issues, such as a camera settings changing accidentally to have lower exposure.
+
+So Why Is Drift So Important?
+-----------------------------
+
+Machine learning models are meant to predict on unseen data, based on previous known data (more accurately, this refers
+to supervised machine learning models, for which drift is most relevant).
+If the data, or the relationships between the different variables and target label, has changed, our model may not be as
+accurate as it was before.
+
+Detecting drift is an important warning sign that our model may be not as accurate on the new data compared to the training data, and that it should be
+adjusted or retrained on different data.
+In production environments, detecting drift (and other measured derived from drift, such as model confidence) is often the only way to know that our model performance is deteriorating,
+as in many cases the label is unknown for some time after the prediction is made.
+
+It is important to note that not all changes in data represent drift. For example, periodic changes in data due to daily, weekly or
+seasonal changes are usually not considered drift, as they as they are often present in the training data.
+
+.. image:: /_static/images/general/patterns-of-concept-drift.png
+   :alt: Different change patterns in data
+   :align: center
+
+Different change patterns in data. `Source <https://www.iosrjournals.org/iosr-jce/papers/Vol17-issue1/Version-2/D017122026.pdf>`_.
+
+Which Types of Drift Are There?
+================================
+
+In machine learning, we usually refer to 3 types of drift:
+
+Concept Drift
+-------------
+Concept drift happens when the underline relation between the data and the label has changed. In layman terms, this means that our
+data has changed in a way that affects our label, meaning a model trained on previous data will have difficulty predicting
+on the new data, as the label can't be predicted in the same way anymore.
+
+Concept drift will almost always require some changes to the model, and will require retraining it on newer data.
```

Comment:
Usually this is achieved by retraining of the model on more recent data.

Reply:
Not sure what you meant to comment here. That "recent" is better than "newer"? Or that it's not necessarily on newer data?

Reply:
Should have been more clear on this one. Meant to offer this sentence as a replacement for "and will require retraining it on newer data."

Reply:
"recent" is good but I prefer "newer" because it is in direct contrast to "older". Accepting the general vibe change and will update :)